backend/de.metas.shipper.gateway.dhl/src/main/java/de/metas/shipper/gateway/dhl/DhlDeliveryOrderRepository.java

```diff
@@ -366,51 +302,46 @@ private I_DHL_ShipmentOrderRequest createShipmentOrderRequestPO(@NonNull final D
 					// (2.1) The id column (I_DHL_ShipmentOrder_ID) is used as ShipmentOrder.sequenceNumber since it's unique
 					// nothing to persist here, but must be filled when retrieving the PO
 				}
-			} // fori loop
 		}
 		return shipmentOrderRequest;
 	}
 
 	private void updateShipmentOrderRequestPO(@NonNull final DeliveryOrder deliveryOrder)
 	{
-		for (final DeliveryPosition deliveryPosition : deliveryOrder.getDeliveryPositions()) // only a single delivery position should exist
+		for (final DeliveryOrderLine deliveryOrderLine : deliveryOrder.getDeliveryOrderLines()) // only a single delivery position should exist
```

Comment:
"// only a single delivery position should exist" 

* is this still true? If it is, please, rewrite it with "order line"

Reply:
Theoretically it's true, but it makes no sense currently. There may be more that one DeliveryOrderLine, even though in the old data structure a single DeliveryPosition was expected. Removed misleading comment.